pkg/stages/stages.go

```diff
@@ -423,10 +424,10 @@ func GetCleanupStage(sis values.System, l types.KairosLogger) []schema.Stage {
 			},
 		},
 		{
-			Name: "remove hostname",
+			Name: "truncate /etc/hostname",
```

Comment:
For consistency with the other command names I'd remove the etc part here

```suggestion
			Name: "truncate hostname",
```

Reply:
@mauromorales  fixed